models/fetch-article.model.js

```diff
@@ -0,0 +1,22 @@
+const db = require("../db/connection");
+
+exports.fetchArticle = (article_id) => {
+  if (isNaN(article_id)) {
+    return Promise.reject({
+      message: "This is a bad request, endpoint not found!",
+      status: 400,
+    });
+  }
```

Comment:
This is a good way to do this, but alternatively you can allow the sql query to throw an error for this instead.